src/org/labkey/test/components/experiment/LineageGraph.java

```diff
@@ -0,0 +1,105 @@
+package org.labkey.test.components.experiment;
+
+import org.labkey.test.Locator;
+import org.labkey.test.components.Component;
+import org.labkey.test.components.WebDriverComponent;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+public class LineageGraph extends WebDriverComponent<LineageGraph.ElementCache>
+{
+    final WebElement _el;
+    final WebDriver _driver;
+
+    public LineageGraph(WebElement element, WebDriver driver)
+    {
+        _el = element;
+        _driver = driver;
+    }
+
+    /**
+     * finds the list of edges/nodes to the one currently focused, by name
+     * @param listTitle
+     * @return
+     */
+    public NodeDetails getDetails(String listTitle)
+    {
+        return elementCache().summaryList(listTitle);
+    }
+
+    /**
+     * finds all details lists in the details panels
+     * @return
+     */
+    public List<NodeDetails> getDetails()
+    {
+        return elementCache().summaryLists();
+    }
+
```

Comment:
The attributes of the sample, or whatever has the focus, are also shown in the right hand panel next to the canvas/visgraph. I don't see how I get those attributes in this control. There is also an "Overview" and "Lineage" link at the top of the panel as well (next to the items name).

Reply:
I've added a function to the top-level component that will return the data in the table as a map.

Worth noting that it's in a perfectly good responsiveGrid that we use in lots of places to show key/value pairs, but which doesn't have this utility (of extrcting columnName/value).  Think we should extend responsiveGrid with a special subType that will pull keys/values out of this kind of table